Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Controllers/InvokeApiController.cs

```diff
@@ -0,0 +1,239 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.TestTool.Runtime;
+using Amazon.Lambda.TestTool.Runtime.LambdaMocks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Amazon.Lambda.TestTool.BlazorTester.Controllers
+{
+    [Route("[controller]")]
+    public class InvokeApiController : ControllerBase
+    {
+        private readonly LocalLambdaOptions _lambdaOptions;
+        private LambdaConfigInfo _lambdaConfig;
+
+        public InvokeApiController(LocalLambdaOptions lambdaOptions)
+        {
+            _lambdaOptions = lambdaOptions;
+        }
+
+        [HttpPost("execute")]
+        public async Task<IActionResult> ExecuteFunction()
+        {
+            if (!TryGetConfigFile(out var lambdaConfig))
+            {
+                return StatusCode((int)HttpStatusCode.InternalServerError,
+                    new InternalException("ServiceException", "Error while loading function configuration"));
+            }
+
+            if (lambdaConfig.FunctionInfos.Count == 0)
+            {
+                return NotFound(new InternalException("ResourceNotFoundException", "Default function not found"));
+            }
+
+            return Ok(await ExecuteFunctionInternal(lambdaConfig, lambdaConfig.FunctionInfos[0]));
+        }
+
+        [HttpPost("execute/{functionName}")]
```

Comment:
Can we just have the `2015-03-31/functions/{functionName}/invocations` path. Somebody I would like this tool to have a sort of API Gateway emulator and I want to avoid any possible name collisions between the user's defined resource path to any of our resource paths.

Reply:
See my comments below ⬇️